src/controllers/chess-game-controller.js

```diff
@@ -194,30 +276,40 @@ export default class ChessGameController {
     game.doMove(game.getMoves(BP5)[1]);
   }
 
+  rotateBoard() {
+    if (this.game.getActivePlayer() !== this.game.whitePlayer) {
+      this.board.setAttribute("rotate", "false");
+    } else {
+      this.board.setAttribute("rotate", "true");
+    }
+  }
+
   mountPieces() {
     this.mountPlayerPieces(this.game.whitePlayer);
     this.mountPlayerPieces(this.game.blackPlayer);
   }
 
   mountPlayerPieces(player) {
     for (let piece of player.livePieces) {
-      const squareElement = this.board.getSquare(piece.row, piece.file);
-      const chessPieceElement = document.createElement("chess-piece");
-      chessPieceElement.piece = piece;
-
-      chessPieceElement.addEventListener(
-        "click",
-        this.handlePieceClick.bind(this)
-      );
-      squareElement.appendChild(chessPieceElement);
+      this.mountSinglePiece(piece);
     }
   }
 
+  mountSinglePiece(piece) {
+    const squareElement = this.board.getSquare(piece.row, piece.file);
+    const chessPieceElement = document.createElement("chess-piece");
+    chessPieceElement.piece = piece;
+    chessPieceElement.addEventListener(
+      "click",
+      this.handlePieceClick.bind(this)
+    );
+    squareElement.appendChild(chessPieceElement);
+  }
+
```

Comment:
This method reads nicely. Simple, single responsibility, descriptive variable names that adhere to conventions